docs/docs/process-payments-with-stripe.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Process Payments with Stripe
+---
+
+## Why Stripe and Gatsby?
+
+Gatsby is an excellent choice for an e-commerce website because static sites are secure, incredibly fast, and easy to host. Whether you're running an online store, accepting donations for a charity, or billing based on usage, you'll need a way to accept payments. Payment processing platforms like Paypal and Stripe are ideal solutions for these use cases.
+
+Stripe is a developer-friendly payment processing platform. With a Stripe account, you can process payments, create recurring payments, and send invoices. They offer many more products which you can explore on [the Stripe website](https://stripe.com/). Stripe provides an excellent dashboard to manage your account as well as a high-quality, well-documented API. The [Stripe API documentation](https://stripe.com/docs/api) is an excellent resource.
+
+## Setting up your Stripe account
```

Comment:
Can we title this "Prerequisites"? You've done a wonderful job of explaining the "how and why" and I'd like to draw a line between this doc (the following few paragraphs specifically) and the tutorial. This might be a good place to link to [Making an e-commerce Gatsby Site with Stripe](https://www.gatsbyjs.org/tutorial/ecommerce-tutorial/) for more detailed instructions.

Reply:
Yes, I'll adjust that.